models/order.js

```diff
@@ -25,18 +25,11 @@ orderSchema.statics.getOrders = function() {
 orderSchema.statics.updateEntry = function(orderId, entryIndex, update) {
   return this.findOne({ id: orderId })
     .then((order) => {
```

Comment:
Yeah, I tried using those in some of the places where we chain a lot of `then`s, but I think the ramda functions require every argument to return a promise, so we'd have to explicitly wrap the ones where we are just doing some transforms (like `toEntry` or something).